examples/passing/PolyLabels.purs

```diff
@@ -0,0 +1,65 @@
+module Main where
+
+import Prelude
+import Control.Monad.Eff
+import Control.Monad.Eff.Console
+import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
+
+foreign import unsafeGet
+  :: forall r a
+   . String
+  -> Record r
+  -> a
+
+foreign import unsafeSet
+  :: forall r1 r2 a
+   . String
+  -> a
+  -> Record r1
+  -> Record r2
+
+get
+  :: forall r l a
+   . IsSymbol l
+  => Lookup l r a
+  => SProxy l
+  -> Record r
+  -> a
+get l = unsafeGet (reflectSymbol l)
+
+set
```

Comment:
@LiamGoodacre Yes, but I think I prefer to modify `Lookup` so that it also returns the tail (the row with the chosen label removed). That way, inference works both ways - we can view it as subtracting the label from one argument, or adding it to the other (both functional dependencies are valid).